business/repository/dao/queries.go

```diff
@@ -48,6 +48,100 @@ const (
 	WHERE
 		"row_num" = 1;
 	
+`
+
+	FetchFDHighIR = `WITH RankedPlans AS (
```

Comment:
Expecting single query which will give 8 rows. All min and max rates of fsi

Reply:
done